css-highlight-api-1/Overview.bs

```diff
@@ -450,6 +461,26 @@ Priority of Overlapping Highlights</h4>
 		</div>
 	</div>
 
+<h4 id=highlight-types>
+Highlight types</h4>
+
+	A [=custom highlight=]'s {{Highlight/type}} attribute is used by authors to specify the semantic
+	meaning of the highlight. This allows assistive technologies to include this meaning when
+	exposing the highlight to users.
+
+	A custom highlight will have a default type of {{HighlightType/highlight}}
+	if its {{Highlight/type}} attribute has not been explicitly set.
+
+	Note: Authors should set a [=custom highlight=]'s {{Highlight/type}} to
+	{{HighlightType/spelling-error}} when that [=custom highlight=] is being used to emphasize
+	misspelled content. Authors should set a [=custom highlight=]'s {{Highlight/type}} to
+	{{HighlightType/grammar-error}} when that [=custom highlight=] is being used to emphasize
+	content that is grammatically incorrect. For all other use cases {{Highlight/type}} should
+	be left as {{HighlightType/highlight}}.
```

Comment:
Should we also add some comments about what UAs should do with the type?

Reply:
Added some normative text and another Note about this.